suites/quincy/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -362,4 +353,12 @@ tests:
         tiebreaker_mon_site_name: arbiter
         delete_pool: true
       desc: Test the cluster when the Data site is moved to maintenance mode
-      abort-on-fail: true
+
+  - test:
+      name: Negative scenarios - post-deployment
+      module: test_stretch_negative_scenarios.py
+      polarion-id: CEPH-83584499
```

Comment:
Why the steps are not appended to the CEPH-83573621 test case? After configuring the stretch cluster  execute the negative scenarios

Reply:
New test has been created for -ve scenarios, and not added in existing test case for Stretch mode deployment.